src/NLog/Targets/ConsoleWordHighlightingRule.cs

```diff
@@ -116,7 +130,7 @@ public Regex CompiledRegex
                         }
                     }
 
-                    RegexOptions regexOptions = RegexOptions.Compiled;
+                    RegexOptions regexOptions = RegexOptions.None;
```

Comment:
Is this a performance boost according to the test? 

Because we also won't use regex caching, (static regex), which is maybe not that nice.

Also I'm bit in doubt with this change due to the name of the property. What do you think?